internal/expr/parts.go

```diff
@@ -32,7 +32,8 @@ func (fn fullName) String() string {
 // inputPart represents a named parameter that will be sent to the database
 // while performing the query.
 type inputPart struct {
-	source fullName
+	source    fullName
+	rawString string
```

Comment:
The 'String' suffix on a string-type field seems redundant, perhaps this could just be `raw`?